Formula/mesa.rb

```diff
@@ -6,6 +6,7 @@ class Mesa < Formula
   url "https://mesa.freedesktop.org/archive/mesa-21.3.7.tar.xz"
   sha256 "b4fa9db7aa61bf209ef0b40bef83080999d86ad98df8b8b4fada7c128a1efc3d"
   license "MIT"
+  revision 1
```

Comment:
It looks like this change doesn't affect usage of `mesa`, so we can probably drop the `revision` bump.